src/index.scss

```diff
@@ -37,7 +37,7 @@ a {
 
 input,
 button {
-  outline: none;
+  outline: none !important;
   transition: 0.2s;
```

Comment:
Why is there a transition on the `input`?

Reply:
I wanted to add it only to the button but then I thought that it won't be a bad thing to do if I also add it to the input, because if there are any effects written for the inputs they will look smoother.